scripts/algorithms/m-svm-predict.dml

```diff
@@ -59,7 +60,7 @@ scores = X %*% W[1:m,] + ones %*% b;
 if(cmdLine_scores != " ")
 	write(scores, cmdLine_scores, format=cmdLine_fmt);
 
-if(cmdLine_Y != " "){
+if(cmdLine_scoring_only == FALSE){
 	y = read(cmdLine_Y);
```

Comment:
in order to make these scoring scripts friendlier for mlcontext and jmlc it would be good to rename y to Y (to make it consistent with the command line argument because user have to bind there in-memory inputs against these left-hand-side variables)